Functions.Templates/Templates/DurableFunctionsOrchestration-CSharp-Isolated/.template.config/template.json

```diff
@@ -0,0 +1,62 @@
+{
+  "author": "Microsoft",
+  "classifications": [
+    "Azure Function",
+    "Durable Functions Orchestration"
+  ],
+  "name": "DurableFunctionsOrchestration",
+  "identity": "Azure.Function.CSharp.Isolated.DurableFunctionsOrchestration.1.x",
+  "groupIdentity": "Azure.Function.DurableFunctionsOrchestration",
+  "shortName": "durable",
+  "tags": {
+    "language": "C#",
+    "type": "item"
+  },
+  "sourceName": "DurableFunctionsOrchestrationCSharp",
+  "symbols": {
+    "namespace": {
+      "description": "namespace for the generated code",
+      "replaces": "Company.Function",
+      "type": "parameter"
+    },
+    "HostIdentifier": {
+      "type": "bind",
+      "binding": "HostIdentifier"
+    }
+  },
+  "primaryOutputs": [
+    {
+      "path": "DurableFunctionsOrchestrationCSharp.cs"
+    }
+  ],
+  "defaultName": "DurableFunctionsOrchestrationCSharp",
+  "postActions": [
+    {
+      "Description": "Adding Reference to Microsoft.Azure.Functions.Worker.Extensions.DurableTask NuGet package",
```

Comment:
Should "Reference" be lowercased here?

Reply:
I also noticed the "Description" is capitalized here but lowercase in some of the other post actions.

Reply:
Lowercased "reference". Left "Description" as is, as that is consistent with all other `template.json` files and with the other properties in this section.